src/utils/displayFunctions.tsx

```diff
@@ -61,14 +61,21 @@ const getLogoForBrandPetname = (brandPetname: string) => {
 export const displayPetname = (pn: Array<string> | string) =>
   Array.isArray(pn) ? pn.join('.') : pn;
 
+const DEFAULT_DECIMAL_PLACES = 6;
+
 export const makeDisplayFunctions = (brandToInfo: Map<Brand, BrandInfo>) => {
-  const getDecimalPlaces = (brand: Brand) => {
+  const getDecimalPlaces = (brand?: Brand) => {
+    if (!brand) return DEFAULT_DECIMAL_PLACES;
+
     // XXX for rpc brands that don't come in the purse watcher
     if ('boardId' in brand) {
       const { boardId } = brand as unknown as { boardId: string };
       const info = wellKnownBrands[boardId];
-      assert(info, `unknown boardId ${boardId}`);
```

Comment:
This was throwing when trying to parse the IST brand. I guess my local chain has a different board id for IST now than what's in `wellKnownBrands`. I figure switching to a warning is good enough since it should still work on mainnet.

Reply:
thanks. I hit this recently and wasn't sure why. 

Could we make this app able to deal with both board ids? Or more comprehensively, both `agoricNames` configs?

Reply:
I think hardcoding to 6 is pretty safe for this case. The problem is I don't think the board id can be relied on to never change for local chains. So, it'd be better to read `agoricNames.vbankAsset`, which doesn't exist on mainnet currently, so we'd have to fallback to a hardcoded value anyway.